packages/strapi-admin/index.js

```diff
@@ -9,6 +10,64 @@ const chokidar = require('chokidar');
 
 const getPkgPath = name => path.dirname(require.resolve(`${name}/package.json`));
 
+function getCustomWebpackConfig(dir, config) {
+  const adminConfigPath = path.join(dir, 'admin', 'admin.config.js');
+  let webpackConfig = _.cloneDeep(getWebpackConfig(config));
+
+  if (fs.existsSync(adminConfigPath)) {
+    const adminConfig = require(path.resolve(adminConfigPath));
+
+    if (_.isFunction(adminConfig.webpack)) {
+      webpackConfig = adminConfig.webpack(webpackConfig, webpack);
```

Comment:
You should throw an error if the call returns undefined and add a nice log message

Reply:
Yes absolutely!